client/landing/stepper/declarative-flow/internals/steps-repository/choose-a-domain/style.scss

```diff
@@ -1,8 +1,10 @@
 $videopress-mobile-layout-width: 1200px;
+$videopress-link-color: #c377ff;
+$videopress-background: #010101;
 
 .videopress.choose-adomain {
 	--color-text-subtle: #a7aaad;
-	--studio-green-60: #1ed15a;
+	--studio-green-60: #c377ff;
```

Comment:
Seems kinda silly to override the `green` color - but was easiest solution to update everywhere there was a green color displaying.

Reply:
Maybe leave a comment here to explain whats happening, so someone doesn't try to fix it?